cvFit/src/app/exercises/exercise-detail.component.html

```diff
@@ -0,0 +1,44 @@
+
+<div class='card'>
+    <div class='card-header'>
+      {{pageTitle}}
+    </div>
+  
+    <div class='card-body'
+         *ngIf='exercise'>
+      <div class='row'>
+        <div class='col-md-8'>
+          <div class='row'>
+            <div class='col-md-4'>Name:</div>
+            <div class='col-md-8'>{{exercise.exerciseName}}</div>
+          </div>
+          <div class='row'>
+            <div class='col-md-4'>Description:</div>
+            <div class='col-md-8'>{{exercise.description}}</div>
+          </div>
+          <div class='row'>
+            <div class='col-md-4'>Tutorial: {{exercise.video}}</div>
+            <iframe width="420" height="315"
+              src="https://www.youtube.com/embed/tgbNymZ7vqY?autoplay=1&mute=1">
+            </iframe>
+          </div>
+        </div>
+  
+        <div class='col-md-4'>
+          <img class='center-block img-responsive'
+               [style.width.px]='200'
```

Comment:
All of your styling should be done in CSS. inline styling is very rare to see. HTML will represent the page structure whereas CSS will represent how that structure is styled.

Reply:
I won't hold this PR up for this but it is something you should go back and remove for anywhere you may be adding inline styling.